nifi-system-tests/nifi-system-test-suite/src/test/java/org/apache/nifi/tests/system/clustering/FlowSynchronizationIT.java

```diff
@@ -399,6 +400,10 @@ public void testCannotJoinClusterIfMissingNar() throws NiFiClientException, IOEx
 
         // Wait for node to show as disconnected because it doesn't have the necessary nar
         waitForNodeState(2, NodeConnectionState.DISCONNECTED);
+
+        // Reconnect so that flow teardown can happen
+        reconnectNode(2);
+        waitForAllNodesConnected();
```

Comment:
This results in test failures on both automated and local builds, which may indicate some other problem with this test method.